inline pgp messages were displayed incorrectly when pgp_auto_decode=yes, see
upstream http://bugs.mutt.org/3269

--- a/crypt-gpgme.c
+++ b/crypt-gpgme.c
@@ -2371,16 +2371,14 @@
               safe_fclose (&pgpout);
             }
         }
-#if 0
       else
       {
-        /* why would we want to display this at all? */
+        /* A traditional PGP part may mix signed and unsigned content */
         /* XXX - we may wish to recode here */
         if (s->prefix)
           state_puts (s->prefix, s);
         state_puts (buf, s);
       }
-#endif
     }
 
   m->goodsig = (maybe_goodsig && have_any_sigs);
